CMakeLists.txt

```diff
@@ -97,16 +97,27 @@ endif()
 ################################################################################
 # Global configuration types
 ################################################################################
-if (CMAKE_SYSTEM_NAME STREQUAL "NintendoSwitch")
-set(CMAKE_C_FLAGS_DEBUG "-g -ffast-math -DDEBUG")
-set(CMAKE_CXX_FLAGS_DEBUG "-g -ffast-math -DDEBUG")
-set(CMAKE_C_FLAGS_RELEASE "-O3 -ffast-math -DNDEBUG")
-set(CMAKE_CXX_FLAGS_RELEASE "-O3 -ffast-math -DNDEBUG")
-else()
-set(CMAKE_C_FLAGS_RELEASE "-O2 -DNDEBUG")
-set(CMAKE_CXX_FLAGS_RELEASE "-O2 -DNDEBUG")
-set(CMAKE_OBJCXX_FLAGS_RELEASE "-O2 -DNDEBUG")
-endif()
+set(COMMON_FLAGS "-Wall -Wextra -Wno-error \
+          -Wno-return-type \
+          -Wno-unused-parameter \
+          -Wno-unused-function \
+          -Wno-unused-variable \
+          -Wno-missing-field-initializers \
+          -Wno-parentheses \
```

Comment:
by memory msvc need different kind of flag

Reply:
and by seing ci in your branch it fail for windows https://github.com/V10lator/SpaghettiKart/actions/runs/15885724756/job/44797309200